Sources/RawStructuredFieldValues/FieldSerializer.swift

```
@@ -204,6 +204,15 @@ extension StructuredFieldValueSerializer {
             self.data.append(asciiQuestionMark)
             let character = bool ? asciiOne : asciiZero
             self.data.append(character)
+        case .date(let date):
+            self.data.append(asciiAt)
+
+            // Then, serialize as integer.
+            guard let wideInt = Int64(exactly: date), validIntegerRange.contains(wideInt) else {
+                throw StructuredHeaderError.invalidIntegerOrDecimal
```

Probably better to throw `invalidDate` here.